_includes/snippets/ep02/runjob.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -x
```

Comment:
Is this file needed now? Mentioned twice in 02-benchmark-and-scaling. Or can `{{ site.stuff }}` be used in runjob.sh and be run as if it were PBS or SBATCH. Do the abstractions make that transition?